### DISCUSSION TITLE
How to apply augmentation to masks or labels

### DISCUSSION THREAD

**Discussion by gtgrp-user** [torchgeo/torchgeo#1417] [Q&A] [answered]
Hi,

I have successfully applied the following transforms to my dataloader for the training images. However, I'm unsure how to apply the random horizontal and vertical flips to the masks/labels.

```
transforms = AugmentationSequential(
    MinMaxNormalize(mins, maxs),
    indices.AppendNDWI(index_green=1, index_nir=3),
    indices.AppendNDVI(index_nir=3, index_red=2),
    K.RandomHorizontalFlip(p=0.5),
    K.RandomVerticalFlip(p=0.5),
    data_keys=["image"],
)
```

**Comment by isaaccorley** [comment, marked as answer]
If you modify data_keys to `data_keys=["image", "mask"]` this will apply the correct transforms to both the image and mask. `AugmentationSequential` is simply a custom wrapper around `kornia.augmentation.AugmentationSequential` if you're curious about further documentation.
